app/helpers/timer_sessions_helper.rb

```diff
@@ -3,7 +3,7 @@
 module TimerSessionsHelper
   MAX_SUBJECT_LENGTH = 75
   SECONDS_IN_MINUTE = 60
-  GAP_LIMIT_IN_MINUTES = 5
+  GAP_LIMIT_IN_MINUTES = 1
```

Comment:
Meaning it's no longer a gap than 5 minutes? I'd like that to be infinite.

Reply:
No its the other way around. If the gap is longer than that number it counts as a gap. Having a 1 minute gap marked makes more sense to me than only counting it form 5 minutes upwards

Reply:
What about no gap?

Reply:
Its a `>=`, which means as soon as the gap is one minute it shows. Making it less means everywhere has a gap